packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -98,6 +98,10 @@ gem build %{gem_name}.gemspec
 %{?scl:EOF}
 
 %install
+%if 0%{?fedora} || 0%{?rhel} >= 8
+sed -i 's|#!/usr/bin/env python|#!/usr/bin/python3|g' sbin/upload-salt-reports
+%endif
```

Comment:
I think this needs to happen in the build phase. You're now modifying the source files, but in %install it's already `%{gem_instdir}/sbin/upload-salt-reports`:
```
*** ERROR: ambiguous python shebang in /usr/share/gems/gems/smart_proxy_salt-3.1.1/sbin/upload-salt-reports: #!/usr/bin/env python. Change it to python3 (or python2) explicitly.
*** ERROR: ambiguous python shebang in /usr/sbin/upload-salt-reports: #!/usr/bin/env python. Change it to python3 (or python2) explicitly.
```

Reply:
Yeah, I noticed late at night it didn't have the desired effect. I'll have to spin up an environment for buidling packages to reduce rtt for this

Reply:
You can also submit a PR with just salt to get quicker testing.

Reply:
:+1: If you want to just pull salt out of this PR and put it on its own we can get the bulk of the changes in and you can iterate faster.